src/containers/WithQAing.jsx

```diff
@@ -219,11 +157,7 @@ export const WithQAing = ComposedComponent => {
       const contentFunc = this.answerAccessorGets[answerAccessor];
 
       if (contentFunc) {
-        return contentFunc(data) || data;
-      }
-
-      if (includes(answerAccessor, 'range')) {
```

Comment:
I think this was my own oddity, instead of using the mapped object I did this manual `if`, moved `range` into the mapped object of answer accessors